### ISSUE TITLE
autofocus only works if Editor is created with a mobiledoc

### ISSUE THREAD

**Issue by gdub22** [bustlelabs/mobiledoc-kit#511]
This currently will not autofocus, because there is no document, therefore nowhere to place the cursor:
```js
const editor = new Mobiledoc.Editor({
  placeholder: 'Write here...',
  autofocus: true
})
```

This works:
```js
const editor = new Mobiledoc.Editor({
  placeholder: 'Write here...',
  autofocus: true,
  mobiledoc: {
    version: '0.3.0',
    markups: [],
    atoms: [],
    cards: [],
    sections: [[1, "p", [[0, [], 0, ""]]]]
  }
})
```

Auto-creating a blank mobiledoc if not provided may have some unwanted side effects.
Perhaps offering a public constant to a blank doc could be a compromise:
```js
const editor = new Mobiledoc.Editor({
  placeholder: 'Write here...',
  autofocus: true,
  mobiledoc: Mobiledoc.BlankMobiledoc
})
```

**Comment by mixonic** [comment]
Agreed, added this to the [draft features 1.0 doc](https://docs.google.com/document/d/1Tn3XdADu50Q0VStF5sIeUqlEDWqRJRX5-ZFwUKJ4ZS8/edit), will make a milestone or something today.

**Comment by knownasilya** [comment]
Also ran into this issue.

**Comment by sdhull** [comment]
Minor nit: autofocus only works if Editor is created with a mobiledoc _with sections._ That is to say, the [`EMPTY_MOBILEDOC` in ember-mobiledoc-editor](https://github.com/bustle/ember-mobiledoc-editor/blob/master/addon/components/mobiledoc-editor/component.js#L23) will not suffice.

**Comment by sdhull** [comment]
FWIW, seems like this fix doesn't work in Safari. It's unclear if it's due to something specific in our implementation or a more general problem.
